extensions/2.0/Khronos/KHR_lights_environment/README.md

```diff
@@ -34,24 +43,21 @@ Secondly, it ensures that rendering of the image-based lighting is consistent ac
 A conforming implementation of this extension must be able to load the image-based environment data and render the PBR materials using this lighting.  
 
 The environment light is defined by a cubemap, this is to simplify realtime implementations as these are likely to use cubemap texture format.  
-Cubemaps can be supplied pre-filtered, by including the pre-filtered mip-levels, where each mip-level corresponds to increasing roughness values.
-If cubemap does not include mip-levels then filtering will be performed at load time.
-If a compressed texture format is used then pre-filtered mip-levels must be included.  
+Cubemaps shall be supplied without pre-filtered mip-maps for roughness values > 0, pre-filtering shall be done by the client.  
+[See Specular radiance cubemaps](#specular-radiance-cubemaps)  
+
+If a compressed texture format is used then pre-filtered mip-levels for roughness values > 0 shall be specified.  
```

Comment:
> "If a compressed texture format is used then pre-filtered mip-levels for roughness values > 0 shall be specified."

This is undesired in my opinion.  I would prefer that no pre-filtered mip-levels are provided for both compressed and non-compressed texture formats.

Reply:
I see your point @bhouston - I just think there is a risk of confusion.
Sure, it's perfectly possible to sample the compressed textures to create filtered mips.
However, since there is (most likely) no compression capability on the target - the filtered mips will be in a non compressed format.
Implementations will end up with more uniform samplers and the filtered mips will be in a non compressed format, requiring more GPU memory.
Sure, the mips are smaller in size but it would still add some 30% or the original uncompressed size - likely to be larger than the compressed mip-level 0.